packages/roosterjs-editor-plugins/lib/plugins/CustomReplace/CustomReplace.ts

```diff
@@ -99,6 +99,7 @@ export default class CustomReplacePlugin implements EditorPlugin {
 
         const searcher = this.editor.getContentSearcherOfCursor(event);
         const stringToSearch = searcher.getSubStringBefore(this.longestReplacementLength);
+        const sourceTag = range.commonAncestorContainer.parentElement.tagName;
```

Comment:
commonAncestorContainer can be different kind of elements, and getting its parent element makes this even more unpredictable. Why we need to do this?

Reply:
I need to get the format of the current string, specifically if it is superscripted.

Reply:
Can a case like that happen in roosterjs?

Reply:
why not?

Reply:
user can paste anything, so any kind of those html can happen